src/taipy/gui/gui.py

```diff
@@ -175,17 +175,21 @@ class Gui:
     __UI_BLOCK_NAME = "TaipyUiBlockVar"
     __MESSAGE_GROUPING_NAME = "TaipyMessageGrouping"
     __ON_INIT_NAME = "TaipyOnInit"
-    __CONTENT_ROOT = "/taipy-content/"
-    __UPLOAD_URL = "/taipy-uploads"
-    _EXTENSION_ROOT = "/taipy-extension/"
+    __ARG_CLIENT_ID = "client_id"
+    __INIT_URL = "taipy-init"
+    __JSX_URL = "taipy-jsx"
+    __CONTENT_ROOT = "taipy-content"
+    __UPLOAD_URL = "taipy-uploads"
+    _EXTENSION_ROOT = "taipy-extension"
+    __USER_CONTENT_URL = "taipy-user-content"
     __SELF_VAR = "__gui"
     __DO_NOT_UPDATE_VALUE = _DoNotUpdate()
 
     __RE_HTML = re.compile(r"(.*?)\.html")
     __RE_MD = re.compile(r"(.*?)\.md")
     __RE_PAGE_NAME = re.compile(r"^[\w\-\/]+$")
 
-    __reserved_routes: t.List[str] = ["taipy-init", "taipy-jsx", "taipy-content", "taipy-uploads"]
+    __reserved_routes: t.List[str] = [__INIT_URL, __JSX_URL, __CONTENT_ROOT, __UPLOAD_URL, _EXTENSION_ROOT, __USER_CONTENT_URL]
     _aggregate_functions: t.List[str] = ["count", "sum", "mean", "median", "min", "max", "std", "first", "last"]
```

Comment:
This could be better of in `pandas_data_accessor` don't you think?

Reply:
why ?
This is used to block the creation of pages that would overlap ...

Reply:
The '_aggregate_functions' part